testdata/p4_16_samples/action_call_ebpf.stf

```diff
@@ -1,16 +1,17 @@
 # bit<32> A bit<32> B
 # In the output B = (A + 10)
 
-packet 0 00000000 ABCDEF01
-expect 0 00000000 ABCDEF01
+packet 0 00000000 00000000 00000000 00000000 00000000 ABCDEF01
```

Comment:
I have not been able to find a way to push packets that do not conform to the Ethernet minimum size, so I have to append a bogus header to each test packet. Otherwise, sendto() justs rejects it. I am sure it is possible, but I am unsure which combination might work best. Maybe packet_mmap may help here.